### PR TITLE
GO 3: Conclusion

### PR DESCRIPTION
**Pull request by fajar13k**
- End of Game_Osu 3 Tourney
- Add GO 3 winners to Tournaments index

---

### CODE REVIEW

File: wiki/Tournaments/GO/3/en.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ tags:
 
 ![GO 3 Logo](img/logo.png)
 
-The **Game\_Osu Tourney 3** (**GO 3**) is a [Logitech](https://www.logitech.com/ "Logitech website") sponsored worldwide one-on-one osu! tournament hosted by ![][flag_FR] [0xy](https://osu.ppy.sh/users/9239673) and ![][flag_FR] [Kaanibaru](https://osu.ppy.sh/users/3117824), and supported by ![][flag_FR] [Sardoche](https://osu.ppy.sh/users/4383702). It is the third installment of the Game\_Osu Tourney.
+The **Game_Osu Tourney 3** (***GO 3***) was a [Logitech](https://www.logitech.com/ "Logitech website") sponsored worldwide one-on-one osu! tournament hosted by ![][flag_FR] [0xy](https://osu.ppy.sh/users/9239673) and ![][flag_FR] [Kaanibaru](https://osu.ppy.sh/users/3117824), and supported by ![][flag_FR] [Sardoche](https://osu.ppy.sh/users/4383702). It was the third installment of the Game_Osu Tourney.
```
Comment (TPGPL) [inline wiki/Tournaments/GO/3/en.md]: not escaping the `_` here?

Comment (cl8n) [reply wiki/Tournaments/GO/3/en.md]: there's nothing to escape, underscore doesn't do anything here

Comment (fajar13k) [reply wiki/Tournaments/GO/3/en.md]: So change it or no?

Comment (Joehuu) [reply wiki/Tournaments/GO/3/en.md]: Leave it as is.

Comment (cl8n) [inline wiki/Tournaments/GO/3/en.md]:
```suggestion
The **Game_Osu Tourney 3** (***GO 3***) was a [Logitech](https://www.logitech.com/ "Logitech website")-sponsored worldwide one-on-one osu! tournament hosted by ![][flag_FR] [0xy](https://osu.ppy.sh/users/9239673) and ![][flag_FR] [Kaanibaru](https://osu.ppy.sh/users/3117824), and supported by ![][flag_FR] [Sardoche](https://osu.ppy.sh/users/4383702). It was the third installment of the Game_Osu Tourney.
```

File: wiki/Tournaments/GO/3/en.md [hunk 2]
```diff
@@ -67,8 +65,9 @@ The Game\_Osu Tourney 3 is run by various community members.
   - [Game\_Osu (EN)](https://www.twitch.tv/gameosu)
   - [Sardoche's Twitch channel](https://www.twitch.tv/sardoche) (Finals and Grand Finals Livestream)
   - [Game\_Osu (FR)](https://www.twitch.tv/gameosu_fr)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vQkxCyBeU6EG2qiHd8ZMrrHYiEm7wxjXCCl73eH8Ttoh7_n9Gz6XfDsm8_tg2Proo9hQR11PATPBGGn/pubhtml)**
 
------------------------
+---
 
 ## Participants
```
Comment (cl8n) [inline wiki/Tournaments/GO/3/en.md]: not sure why flag is its own column in this table, you could do it like everywhere else (Flag + space + Player)

File: wiki/Tournaments/GO/3/en.md [hunk 3]
```diff
@@ -141,31 +140,61 @@ The Game\_Osu Tourney 3 is run by various community members.
...
+- DoubleTime
+  - [kanonxkanon - Calendula Requiem (smallboat) \[Shiki Revive\]](https://osu.ppy.sh/beatmapsets/172293#osu/416492)
+  - [Tsukasa - Heaven's Race Guitar Style (La Cataline, Cherry Blossom) \[Collab\]](https://osu.ppy.sh/beatmapsets/41974#osu/132260)
+  - [FELT - Puppet in the Dark (Mackenzie) \[Out Of Place ARTifactS\]](https://osu.ppy.sh/beatmapsets/326636#osu/725198)
+  - [The Pretty Reckless - Kill Me (Zero\_\_wind) \[Kill!!!\]](https://osu.ppy.sh/beatmapsets/97187#osu/260006)
+- FreeMod
+  - [t+pazolite - Magical Higan Tour 2009 (DJNightmare, Nakagawa-Kanon) \[Collab\]](https://osu.ppy.sh/beatmapsets/40609/#osu/128775)
+  - [5 Step Sound Team - Never Say Die (P o M u T a) \[Die\]](https://osu.ppy.sh/beatmapsets/292599#osu/658488)
+  - [join. SELEN - DADADADADADADA (Sharu) \[Sharu's Extreme\]](https://osu.ppy.sh/beatmapsets/773755#osu/1632692)
+  - [Luschka - Kami no Kotoba (Jounzan) \[Jounzan's Special\]](https://osu.ppy.sh/beatmapsets/817667#osu/1714634)
```
Comment (cl8n) [inline wiki/Tournaments/GO/3/en.md]: still lost as to why the name in parenthesis is anything but the mapset host, all this format does it make it harder to search for the map (picked a random example but plenty of the maps do this)

this format is supposed to go Artist - Title (Creator) [Version], it's misleading to put someone who's not the mapset creator into that field. a computer should be able to make this text

**Pull request merged**